src/app/private/scripts/services/announcementService.js

```diff
@@ -1,188 +1,201 @@
-    return httpCall('/announcement/v1/definitions', req, 'POST')
-  }
+    /**
+     * @method getDefinitions
+     * @desc get announcement type and sender list
+     * @memberOf Services.announcementService
+     * @param {object}  req - rootOrgId, userid and definition type array
+     * @returns {object} returns response of API
+     * @instance
+     */
+    this.getDefinitions = function (req) {
+      return httpCall('/announcement/v1/definitions', req, 'POST')
+    }
 
-		/**
-		 * @method readAnnouncement
-		 * @desc read announcement
-		 * @memberOf Services.announcementService
-		 * @param {object}  req - announcementid, userid
-		 * @returns {object} returns response of API
-		 * @instance
-		 */
-  this.readAnnouncement = function (req) {
-    return httpCall('/announcement/v1/read', req, 'POST')
-  }
+    /**
+     * @method readAnnouncement
+     * @desc read announcement
+     * @memberOf Services.announcementService
+     * @param {object}  req - announcementid, userid
+     * @returns {object} returns response of API
+     * @instance
+     */
+    this.readAnnouncement = function (req) {
+      return httpCall('/announcement/v1/read', req, 'POST')
+    }
 
-		/**
-		 * @method receivedAnnouncement
-		 * @desc received announcement
-		 * @memberOf Services.announcementService
-		 * @param {string}  req - userId, AnnId
-		 * @returns {object} returns response of API
-		 * @instance
-		 */
-  this.receivedAnnouncement = function (userId, AnnId) {
-    var req = {
-      'request': {
-        'userId': userId,
-        'announcementId': AnnId,
-        'channel': 'web'
+    /**
+     * @method receivedAnnouncement
+     * @desc received announcement
+     * @memberOf Services.announcementService
+     * @param {string}  req - userId, AnnId
+     * @returns {object} returns response of API
+     * @instance
+     */
+    this.receivedAnnouncement = function (userId, AnnId) {
+      var req = {
+        'request': {
+          'userId': userId,
+          'announcementId': AnnId,
+          'channel': 'web'
+        }
       }
+      return httpCall('/announcement/v1/received', req, 'POST')
     }
-    return httpCall('/announcement/v1/received', req, 'POST')
-  }
 
-		/**
-		 * @method deleteAnnouncement
-		 * @desc Send requestBody to cancel annoucement API
-		 * @memberOf Services.announcementService
-		 * @param {string}  requestBody - request body
-		 * @returns {object} returns response of API
-		 * @instance
-		 */
-  this.deleteAnnouncement = function (requestBody) {
-    var URL = '/announcement/v1/cancel'
-    return httpCall(URL, requestBody, 'DELETE', {})
-  }
+    /**
+     * @method deleteAnnouncement
+     * @desc Send requestBody to cancel annoucement API
+     * @memberOf Services.announcementService
+     * @param {string}  requestBody - request body
+     * @returns {object} returns response of API
+     * @instance
+     */
+    this.deleteAnnouncement = function (requestBody) {
```

Comment:
methods should take actual params and construct request body -- here and elsewhere.